<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Demo" time="1" tests="0" failures="0" errors="3">
   <testsuite name="Demo" tests="0" failures="0" errors="3" time="1" timestamp="2018-02-22 13:27:43" hostname="bashah - L005401.Apttus.local" id="Test Suites/Demo">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="C:\\Users\\bashah\\Katalon Studio\\TestdataAutomation\\Reports\\Demo\\20180222_132739"/>
         <property name="logFiles" value="C:\\Users\\bashah\\Katalon Studio\\TestdataAutomation\\Reports\\Demo\\20180222_132739\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="bashah - L005401.Apttus.local"/>
         <property name="hostAddress" value="10.13.10.136"/>
         <property name="os" value="Windows 10 64bit"/>
         <property name="katalonVersion" value="5.3.0.1"/>
      </properties>
      <testcase name="Test Cases/Main Test Cases/testdata" classname="Test Cases/Main Test Cases/testdata" status="ERROR">
         <error type="ERROR" message="Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist'"/>
         <system-out><![CDATA[2018-02-22 13:27:44 - [TEST_CASE][ERROR] - Test Cases/Main Test Cases/testdata: Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist'

2018-02-22 13:27:45 - [TEST_STEP][PASSED] - Statement - data = com.kms.katalon.core.testdata.TestDataFactory.findTestData(data): Finding test data with id 'Data Files/data']]></system-out>
         <system-err><![CDATA[2018-02-22 13:27:44 - [TEST_CASE][ERROR] - Test Cases/Main Test Cases/testdata: Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist']]></system-err>
      </testcase>
      <testcase name="Test Cases/Main Test Cases/testdata" classname="Test Cases/Main Test Cases/testdata" status="ERROR">
         <error type="ERROR" message="Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist'"/>
         <system-out><![CDATA[2018-02-22 13:27:45 - [TEST_CASE][ERROR] - Test Cases/Main Test Cases/testdata: Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist'

2018-02-22 13:27:45 - [TEST_STEP][PASSED] - Statement - data = com.kms.katalon.core.testdata.TestDataFactory.findTestData(data): Finding test data with id 'Data Files/data']]></system-out>
         <system-err><![CDATA[2018-02-22 13:27:45 - [TEST_CASE][ERROR] - Test Cases/Main Test Cases/testdata: Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist']]></system-err>
      </testcase>
      <testcase name="Test Cases/Main Test Cases/testdata" classname="Test Cases/Main Test Cases/testdata" status="ERROR">
         <error type="ERROR" message="Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist'"/>
         <system-out><![CDATA[2018-02-22 13:27:45 - [TEST_CASE][ERROR] - Test Cases/Main Test Cases/testdata: Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist'

2018-02-22 13:27:45 - [TEST_STEP][PASSED] - Statement - data = com.kms.katalon.core.testdata.TestDataFactory.findTestData(data): Finding test data with id 'Data Files/data']]></system-out>
         <system-err><![CDATA[2018-02-22 13:27:45 - [TEST_CASE][ERROR] - Test Cases/Main Test Cases/testdata: Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist']]></system-err>
      </testcase>
      <system-out><![CDATA[2018-02-22 13:27:43 - [TEST_SUITE][ERROR] - Demo: Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist']]></system-out>
      <system-err><![CDATA[2018-02-22 13:27:43 - [TEST_SUITE][ERROR] - Demo: Test Cases/Main Test Cases/testdata FAILED because (of) java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/data' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/data' does not exist']]></system-err>
   </testsuite>
</testsuites>
